summaries/sprint11.md

```diff
@@ -0,0 +1,25 @@
+# Sprint 11 retrospective
+
+## Scrum master (Marouane Jaakik)
+
+## Bettens, Louis
+This week I worked on two internal issues within the codebase, namely cleaning
+up our DI modules somewhat, and clarifying the naming of our different
+databases. I spent most of my alloted time on the first task as it required
+more extensive code changes. Overall, this reduced our technical debt.
+
+## Jaakik, Marouane
+
+## Konrad, Jonas Franz
+This week I worked on three tasks. First I added image compression to every image that gets added to the database. This drasticly reduced load times with no noticeable quality loss.
+Then I fixed a bug where downloading a database where not every category has a representative picture set would crash the app. This was caused by the Room database. I hence improved it in that sense as well as making deletions from it cascade correctly. The last task was some various refactorings and adding kdocs in files from the model.
+
+## Lachat, Niels Marco
+This week I worked on making the learning clearer by adding visual feedback. The image to sort now blinks in blue and the target categories turn blue to indicate where to sort the image. Upon sorting, there is a blinking in green or red to give feedback about the sorting. I received some very useful feedback from Jonas and Nicolas, and I am quite happy with how this feature turned out. I spent around 8 hours on the project, including reviews.
+
+## Lenweiter, Martin
+This week I worked on addressing Lint issues. I managed to fix most of them, with a few which we decided to ignore. I spent only about 5 hours of work on this task, which was welcome as I was pretty busy with other classes.
+
+## Vial, Nicolas
+
```

Comment:
```suggestion
## Vial, Nicolas
In this sprint, I simply wrote the KDoc for the dataset editing fragments and did some refactoring following the code review. I worked only 6 hours during this sprint counting the reviews because I was very busy with other projects.
```